src/NLog/Targets/TraceTarget.cs

```diff
@@ -62,6 +63,12 @@ namespace NLog.Targets
     public sealed class TraceTarget : TargetWithLayout
     {
         /// <summary>
+        /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
+        /// </summary>
+        [DefaultValue(false)]
+        public bool WriteLine { get; set; }
```

Comment:
I like IgnoreLogLevel. The link you just provide says that when using TraceInformation, then it converts to TraceEvent with info-log-level. So it actually ignores LogLevel when just using Trace.WriteLine.